.github/workflows/v2.yml

```diff
@@ -1,44 +0,0 @@
-name: Build and Test v2
-
-on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-  push:
-    branches: [main]
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
-  cancel-in-progress: true
-
-jobs:
-  build:
-    name: "Build ${{ matrix.platform }} in ${{ matrix.build_type }}"
-    strategy:
-      matrix:
-        platform: [windows, ubuntu, macos]
-        build_type: [Debug, Release]
-    env:
-      PARALLEL: -j 2
-    runs-on: "${{ matrix.platform }}-latest"
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: "recursive"
-
-      - if: ${{ matrix.platform == 'ubuntu' }}
-        name: Install RandR headers
-        run: |
-          sudo apt-get update
-          sudo apt install xorg-dev libglu1-mesa-dev
-
-      - name: Configure
-        run: cmake -B"build/${{ matrix.platform }}" -DVKB_BUILD_TESTS=ON
-
-      - name: "Build Components ${{ matrix.platform }} in ${{ matrix.build_type }}"
```

Comment:
Can we move these three steps to the existing build pipeline please. That would preserve running unit tests. This should be a fast operation.